### ISSUE TITLE
Disable "execute on application start" PowerShell scripts in safe mode

### ISSUE THREAD

**Issue by bburky** [JosefNemec/Playnite#3121]
### Feature description

It is possible for a poorly behaved "execute on application start" application script to crash Playnite. This is unavoidable with user provided scripts.

A crash during the execution of the "execute on application start" script rightfully triggers the startup error dialog:

> Playnite closed unexpectedly while starting. This is usually caused by 3rd party theme or extension. Do you want to start in safe mode with all 3rd party add-ons disabled?

However, safe mode does not appear to disable the "execute on application start" script. Please disable "execute on application start" scripts when in safe mode, otherwise the user may be stuck in a restart loop depending on what their script does.

**Comment by JosefNemec** [comment]
Can you post example script that can cause this? Script errors should not be able to do this in theory.

**Comment by bburky** [comment]
Never underestimate how silly users (me) can be:
```powershell
 Get-Process -name "Playnite.DesktopApp" | Stop-Process
```

Context: I wrote a short script to detect screen resolution and switch to 1920x1080 on startup (down from 4k) because it runs over Gamestream with lower network bandwidth. Except when lowering the resolution while Playnite is running, it gets slightly blurry (not exactly sure why, presumably something DPI related). ...So I also made the script relaunch Playnite. Was tricky to fix a bug in the script, had to fix it in `config.json`
